pydarn/plotting/rtp.py

```diff
@@ -333,9 +333,33 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
             y = gate2slant(cls.dmap_data[0], y_max)
         time_axis, y_axis = np.meshgrid(x, y)
         z_data = np.ma.masked_where(np.isnan(z.T), z.T)
+        Default = {'noise.sky': (1e0, 1e5),
+          'tfreq': (8, 22),
+          'nave': (0, 60),
+          'p_l': (0, 45),
+          'v': (-200, 200),
+          'w_l': (0, 250),
+          'elv': (0, 45)}
+
+        if np.isinf(zmin) and zmin < 0:
+            zmin = Default[parameter][0]
+            warnings.warn("Warning: zmin is -inf, set zmin to {}. You can"
+                          "set zmin and zmax in the functions"
+                          " options".format(zmin))
+        if np.isinf(zmax) and zmax > 0:
+            zmax = Default[parameter][1]
+            warnings.warn("Warning: zmin is inf, set zmax to {}. You can"
+                          "set zmin and zmax in the functions"
```

Comment:
```suggestion
                          "set zmin and zmax in the function's"
```
I think?

Reply:
```suggestion
                          "set zmin and zmax in the function"
```

I think it's singular since we have 1 function we are using?